### PR TITLE
docs for chromecast_wifi_scan

### PR DESCRIPTION
**Pull request by h00die**
Docs for chromecast_wifi_scan

## Verification

- [x] Start `msfconsole`
- [x] `use auxiliary/scanner/http/chromecast_wifi_scan`
- [x] `info -d`
- [x] **Verify** I didn't mess up any formatting

The info in this is directly from verifying #7746 
@wvu

### CODE REVIEW

**Comment by wvu-r7** [comment]
Corrected the filename in https://github.com/rapid7/metasploit-framework/commit/931c052399eef111a6dd79fc5283ab0121cb73a5. Also, thank you for adding a note about ```(*)```. I neglected to update the module description when I added it.

File: documentation/modules/auxiliary/scanner/http/chromecast_wifi_scan.md [hunk 1]
```diff
@@ -0,0 +1,65 @@
+This module is scanner which enumerates wifi access points visible from a Google Chromecast via its HTTP interface (default port 8080).  Any wifi access point the Chromecast is associated with or can be associated with is marked with an `(*)`.
```
Comment (h00die) [inline documentation/modules/auxiliary/scanner/http/chromecast_wifi_scan.md]: "This module is scanner"... way to go engrish.

Comment (wvu-r7) [inline documentation/modules/auxiliary/scanner/http/chromecast_wifi_scan.md]: Oof.

Comment (wvu-r7) [inline documentation/modules/auxiliary/scanner/http/chromecast_wifi_scan.md]: Fixed in https://github.com/rapid7/metasploit-framework/commit/85912be7a31c69e049340fb2c4e382c6d6c2948e. Thanks.

**Comment by wvu-r7** [comment]
# Release Notes

This adds module docs for ```auxiliary/scanner/http/chromecast_wifi```.

**Pull request merged**